src/app/nav/nav.component.html

```diff
@@ -1,26 +1,18 @@
-<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+<nav class="navbar navbar-expand-md navbar-dark bg-dark ">
   <div class="container">
-    <a class="navbar-brand" href="#">Navbar</a>
-    <button
-      class="navbar-toggler"
-      type="button"
-      data-toggle="collapse"
-      data-target="#navbarColor02"
-      aria-controls="navbarColor02"
-      aria-expanded="false"
-      aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
-    <div class="collapse navbar-collapse" id="navbarColor02">
-      <ul class="navbar-nav mr-auto">
-        <li class="nav-item active">
-          <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
-        </li>
-        <li class="nav-item"><a class="nav-link" href="#">Features</a></li>
-        <li class="nav-item"><a class="nav-link" href="#">Pricing</a></li>
-        <li class="nav-item"><a class="nav-link" href="#">About</a></li>
-      </ul>
-      <app-nav-login></app-nav-login>
-    </div>
+    <a class="navbar-brand" [routerLink]="['/home']" routerLinkActive="router-link-active" >Picture App</a>
+    
+    <ul *ngIf="loggedIn" class="navbar-nav mr-auto">
+      <li class="nav-item"r outerLinkActive="active">
```

Comment:
The space in the item 'r outerLinkActive' is by design or by misteake?

Reply:
fixed by : https://github.com/anowakowski/PictureApp_SPA/pull/11/commits/c3a2049a299fbb179c8eb11fbaa00fe233c94310